scripts/mkimage

```diff
@@ -207,6 +207,7 @@ elif [ "$BOOTLOADER" = "u-boot" ]; then
     echo "image: copying files to part1..."
     mcopy $TARGET_IMG/$IMAGE_NAME.kernel "::/$KERNEL_NAME"
     mcopy $TARGET_IMG/$IMAGE_NAME.system ::/SYSTEM
+    mcopy $RELEASE_DIR/target/*.md5 ::
```

Comment:
This is only copying the `*.md5` files for `u-boot` systems, not for `syslinux` or `bcm2835-bootloader` systems.